commands/Moderation/role.js

```diff
@@ -0,0 +1,38 @@
+exports.run = async (client, msg, [user, role]) => {
+  // Check if the discord server has the role by its name
+  if (msg.guild.roles.exists("name", role)) {
+    // Check if the user already has the role
+    if (user.roles.find("name", role)) {
+      // Remove role if already posses
+      user.removeRole(msg.guild.roles.find("name", role));
+      // Send reply
+      msg.reply(`${user} has lost the ${role} role. :smiley:`);
+    } else {
+      // Add role if the role was missing
+      user.addRole(msg.guild.roles.find("name", role));
+      // Send reply
+      msg.reply(`${user} has been given the ${role} role. :smiley:`);
+    }
+  } else {
+    // Send reply if the role doesnt exist on the server
+    msg.reply(`I am sorry but ${role} role does not exist. :cry:`);
+  }
+};
+
+exports.conf = {
+  enabled: true,
+  runIn: ["text"],
+  aliases: ["addrole", "ar", "removerole", "rr"],
+  permLevel: 2,
```

Comment:
Due to the nature of this command, you should change the level to 3. What if the bot's role is above everyone and a moderator asks the bot to grant them the administrator role? This command has (certainly) a lot of flaws, a set of configured roles (claimable roles) would be more useful in most situations.